src/Console/Coverage.php

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
I think having them here is fine for now. If we wish, the next minor release can move them into a subdirectory.